certi_builder/builder.py

```diff
@@ -28,10 +31,42 @@ def generate_certificate(
     for name in names_list:
         image = Image.open(certificate_image_path)
         drawImage = ImageDraw.Draw(image)
-        drawImage.text(location, name, fill=text_color, font=font)
         if is_bold:
-            drawImage.text((x_location + 1, y_location + 1), name, fill=text_color, font=font)
-            drawImage.text((x_location - 1, y_location - 1), name, fill=text_color, font=font)
+            if is_underlined:
+                twidth, theight = drawImage.textsize(name, font=font)
+                lx, ly = location[0], location[1] + theight
```

Comment:
Make sure to name variables properly, in the sense that it should be understandable when anyone reads it.